src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/ArmorTask.java

```diff
@@ -148,4 +151,35 @@ private boolean isRadioactive(Player p, ItemStack item) {
         return false;
     }
 
+    private boolean shouldProtect(HashedArmorpiece[] armors) {
+        int armorCount = 0;
+        boolean first = true;
+
+        String setID = null;
+        for (HashedArmorpiece armor : armors) {
+            Optional<SlimefunArmorPiece> armorPiece = armor.getItem();
+            if (!armorPiece.isPresent()) return false;
+
+            if (armorPiece.get() instanceof CustomProtection) {
+                CustomProtection protectedArmor = (CustomProtection) armorPiece.get();
+
+                if (first) {
+                    if (protectedArmor.requireFullSet()) setID = armorPiece.get().getSetID();
```

Comment:
Is the id even required in the first place?
I think it would make more sense to have a "ProtectionType" argument for this method and check if all 4 protect against this.

Reply:
I wanted to allow people to create sets that only required 1 part for the protection